coalib/processes/Processing.py

```diff
@@ -267,16 +267,17 @@ def get_file_dict(filename_list, log_printer=None, allow_raw_files=False):
     :return:                Reads the content of each file into a dictionary
                             with filenames as keys.
     """
-    file_dict = {}
+    file_dict = FileDict()
     for filename in filename_list:
         try:
-            with open(filename, 'r',
-                      encoding=detect_encoding(filename)) as _file:
-                file_dict[filename] = tuple(_file.readlines())
+            file_dict[filename] = FileFactory(filename)
+            FileFactory(filename).string
```

Comment:
What is this line for ?

Reply:
The old core relied on a `UnicodeDecodeError` in case of raw files. This is just a workaround for that. When we invoke `FileFactory.string` in case of raw files it will throw a `UnicodeDecodeError` and move into the except case.

I will add a comment in the next push.

Reply:
Not done yet